src/functions-and-arrays.js

```diff
@@ -1,19 +1,86 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers (num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num2 > num1) {
+    return num2;
+    } else {
+      return num1, num2;
+    }
+}
+
+maxOfTwoNumbers (166, 8)
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(wordArr) {
+  if (wordArr.length === 0) {
+    return null;
+  } 
```

Comment:
Quick tip - providing the "if" statement is a one liner, then you don't need to add the curly brackets and write everything in one line. 
`if (wordArr.length === 0) return null`